pkgs/servers/jibri/default.nix

```
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchurl, dpkg, jre_headless }:
+
+stdenv.mkDerivation rec {
+  pname = "jibri";
+  version = "8.0-93-g51fe7a2";
+  src = fetchurl {
+    url = "https://download.jitsi.org/stable/${pname}_${version}-1_all.deb";
+    sha256 = "1w78aa3rfdc4frb68ymykrbazxqrcv8mcdayqmcb72q1aa854c7j";
+  };
+
+  dontBuild = true;
+  nativeBuildInputs = [ dpkg ];
+  unpackCmd = "dpkg-deb -x $src debcontents";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,opt/jitsi/jibri,etc/jitsi/jibri}
+    mv etc/jitsi/jibri/* $out/etc/jitsi/jibri/
+    mv opt/jitsi/jibri/* $out/opt/jitsi/jibri/
+
+    # make a tiny wrapper script to launch jibri
+    printf "#!/bin/bash\nexec ${jre_headless}/bin/java -jar $out/opt/jitsi/jibri/jibri.jar \"\$@\"\n" > $out/bin/jibri
```

```suggestion
    echo "#!/bin/bash\nexec ${jre_headless}/bin/java -jar $out/opt/jitsi/jibri/jibri.jar \"\$@\"" > $out/bin/jibri
```
Also I think using makeWrapper is easier here. Let me know if you need help with that.

---

Hm... I'm not too familiar with makeWrapper. I'll check it out.

---

@SuperSandro2000 also, I don't think replacing printf with echo will work. Echo treats \n as a string literal, printf changes that to a newline.